### DISCUSSION TITLE
Support for excluding hidden columns from export (not toggable)

### DISCUSSION THREAD

**Discussion by emcarson** [pxlrbt/filament-excel#184] [Ideas]
I just noticed that Filament Excel doesn't filter out hidden fields from exports.

I managed to implement it by adding the following:

```php
    ExcelExport::make()
        ->fromTable()
        ->except(function (HasTable $livewire) {
            return collect($livewire->getTable()->getColumns())
                ->filter(fn ($column) => $column->isHidden())
                ->map(fn ($column) => $column->getName())
                ->values()
                ->toArray();
        }),
```

It would be nice that the plugin did that by default, or at least implement a method to easily filter them out, like `->exceptHidden()`.

Thank you.

**Comment by pxlrbt** [comment]
Hm. I think it should be the default. Currently we are only filtering `$hidden` columns (from model) and columns that were toggled as hidden.
